### DISCUSSION TITLE
Customize startup notification

### DISCUSSION THREAD

**Discussion by seth100** [hyperjumptech/monika#1068] [Q&A] [answered]
Hi,

I'm running Monika on Docker. When it starts I get a default notification message on the `notifications` providers I set in the config file saying some thing like `Monika is running from xxxxxxxxx` containing some personal info that I'd prefer not to share on the providers I use for notifications. 
Is there a way to customize that message somehow?

Thanks

**Comment by sapiderman** [comment, marked as answer]
Thanks for the feedback. As of now the're no way to customize the startup notificationj. What you're seeing is a probe to the STUN server to check for internet connectivity. You can disable it with the `-s -1` switch, but it'll still probe once at the beginning. We'll take your privacy concern into consideration. Perhaps we can tie it to the `-v verbose` switch to display it or not.

Keep an eye on the repository for updates.

**Comment by arshadmech** [comment]
Hi, 
i am  using monika on VM. i configured my qa servers for monitoring. is there a way to change the message when server went down.
i am getting this message when server goes down "Message: Alert! Connection refused"

**Reply by sapiderman** [reply]
Hi @arshadmech 

May we know what alert assertion and notification channel you are using? In general, alert messages are configured like this:
```yaml
    alerts:
           - assertion: response.status != 200 (or something similar)
             message: Your custom alert here 
```
You can see [the full docs here](https://monika.hyperjump.tech/guides/alerts). However, if Monika is unable to connect to your probe for whatever reason, a default message "**Probe not accessible**" will be sent to the notification channels you've set. This "Probe not accessible" is sent automatically to warn you that probe connectivity has failed. At this time there is no way to customize this message. 

Hope this helps. Let us know if you have any more questions.
